charts/container-object-storage-interface-controller/templates/_helpers.tpl

```diff
@@ -0,0 +1,62 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "container-object-storage-interface-controller.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "container-object-storage-interface-controller.fullname" -}}
```

Comment:
Isn't this already starting out with 46 chars because the base name is `container-object-storage-interface-controller`. I wonder how detailed we all think the name should be. 

Can we be as short as `cosi-controller`?
`object(-store?)-interface`?
`object(-store)?-interface-controller`?

Reply:
I suggest that resource are named `{{ template "container-object-storage-interface-controller.fullname" . }}` (there is one of each kind, so no need to suffix the name